spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestDelete.java

```diff
@@ -322,9 +302,11 @@ public void testDeleteWithoutScanningTable() throws Exception {
         sql("SELECT * FROM %s ORDER BY id", selectTarget()));
   }
 
-  @Test
+  @TestTemplate
   public void testDeleteFileThenMetadataDelete() throws Exception {
-    Assume.assumeFalse("Avro does not support metadata delete", fileFormat.equals("avro"));
+    assumeThat(fileFormat.equalsIgnoreCase("avro"))
```

Comment:
once `fileFormat` is of type `FileFormat`, this check can be changed to `assumeThat(fileFormat).as(..).isNotEqualTo(FileFormat.AVRO)`